### DISCUSSION TITLE
Extension of VDA5050 - M2X initiative

### DISCUSSION THREAD

**Discussion by FranzPaulSigmatek** [VDA5050/VDA5050#367] [Ideas]
We are succesfully using VDA5050 during the application we always have the issue to communicate to machines.
An extension of VDA5050 would be perfect, so you can implement this stack also into a machine.
Example: 
Machine want to call an AGV 
Machine want to talk to AGV (loading unloading) 
Machine is ready for docking etc.

or second example:
Logistic want to talk to TCS 

This would be also a standard and implementation in every type of PLC / controller would be much easier.

**Comment by AntonDueck** [comment]
the Only Focus of VDA5050 is for communication between a Master Control and an AGV/AMR 

 for your topic you may look at:
https://flw.mb.tu-dortmund.de/projektstart-von-m2x-schnittstelle-zur-kommunikation-zwischen-fahrerlosen-transportfahrzeugen-ftf-und-intralogistischen-anlagen-im-ftf-umfeld/

https://github.com/M2X-Project/m2x

Greet Anton

**Comment by SvenFra** [comment]
Hello @FranzPaulSigmatek ,
the VDA 5050 is very successful in the communication with an AGV to the Fleetmanager. For your descibed uses cases we have created the M2X Project. We are working very close to the VDA 5050 working group. Our goal is it to design universally applicable interfaces for communication between the fleet management (FM)/mobile robots (MR) (such as automated guided vehicles (AGVs) or autonomous mobile robots (AMRs)) and other participants in an industrial environment like load handling stations, order generation, peripherals etc. 
The technical structure is close to the VDA 5050. Im glad that Anton shared the links with you.
Here is the GitHub repo, with the public interface of the communication to a machine: https://github.com/M2X-Project/m2x
The documentation: https://m2x-project.github.io/m2x/ 
The linkedIn page: https://www.linkedin.com/showcase/m2x-project/

Reach out if you have questions. The working group is open to everyone.

PS: We will release the second interface in May which is about the communication to a higher system (ERP, WMS) to create a transport order.
